librocksdb_sys/crocksdb/c.cc

```diff
@@ -524,15 +524,18 @@ struct crocksdb_mergeoperator_t : public MergeOperator {
   void* state_;
   void (*destructor_)(void*);
   const char* (*name_)(void*);
-  char* (*full_merge_)(void*, const char* key, size_t key_length,
-                       const char* existing_value, size_t existing_value_length,
-                       const char* const* operands_list,
-                       const size_t* operands_list_length, int num_operands,
-                       unsigned char* success, size_t* new_value_length);
-  char* (*partial_merge_)(void*, const char* key, size_t key_length,
-                          const char* const* operands_list,
-                          const size_t* operands_list_length, int num_operands,
-                          unsigned char* success, size_t* new_value_length);
+  unsigned char* (*full_merge_)(void*, const char* key, size_t key_length,
```

Comment:
This should be left alone. It returns a new value not a boolean pointer.

Reply:
ah, good catch. I misread it.